src/reducers/clusters.js

```diff
@@ -3,7 +3,6 @@ import { actionReducer, removeMissingItems } from './utils'
 
 const initialState = fromJS({
   clusters: {},
-  loadInProgress: true,
```

Comment:
Yay!  Those were on my list to remove as well.

Reply:
It's nice to be on the same page.